disnake/types/appinfo.py

```diff
@@ -73,3 +73,19 @@ class PartialAppInfo(BaseAppInfo, total=False):
 class PartialGatewayAppInfo(TypedDict):
     id: Snowflake
     flags: int
+
+
+class EditAppInfo(TypedDict, total=False):
+    custom_install_url: Optional[str]
+    description: str
+    role_connections_verification_url: Optional[str]
+    install_params: Optional[InstallParams]
+    integration_types_config: Dict[str, ApplicationIntegrationTypeConfiguration]
+    flags: int
+    icon: Optional[str]
+    cover_image: Optional[str]
+    interactions_endpoint_url: Optional[str]
+    tags: Optional[List[str]]
+    event_webhooks_url: Optional[str]
+    event_webhooks_status: str
```

Comment:
This should be an `EventWebhooksStatus = Literal[1, 2, 3]` at the top of the file, not a str. Same thing for the field on line 60, which probably slipped through in the review of the original implementation (see also https://github.com/DisnakeDev/disnake/pull/1237#discussion_r2363775002)

Reply:
https://github.com/DisnakeDev/disnake/pull/1237/commits/fa9d1b4b67d8a7682d2d4e1d6dca38f21640aeaa